packages/components/src/date-time/types.ts

```diff
@@ -29,6 +35,13 @@ export type TimePickerProps = {
 	 * time as an argument.
 	 */
 	onChange?: ( time: string ) => void;
+
+	/**
+	 * If true, the label will only be visible to screen readers.
+	 *
+	 * @default false
+	 */
+	hideLabelFromVision?: boolean;
```

Comment:
If we do end up having to add these props to TimePicker, we will need to omit them from the main `DateTimePicker` props.

```diff
diff --git a/packages/components/src/date-time/types.ts b/packages/components/src/date-time/types.ts
index 5ae93db47d..acf1201c0c 100644
--- a/packages/components/src/date-time/types.ts
+++ b/packages/components/src/date-time/types.ts
@@ -149,7 +149,10 @@ export type DatePickerProps = {
 };
 
 export type DateTimePickerProps = Omit< DatePickerProps, 'onChange' > &
-	Omit< TimePickerProps, 'currentTime' | 'onChange' > & {
+	Omit<
+		TimePickerProps,
+		'currentTime' | 'onChange' | 'hideLabelFromVision' | 'id'
+	> & {
 		/**
 		 * The function called when a new date or time has been selected. It is
 		 * passed the date and time as an argument.

```

Reply:
Fixed at https://github.com/WordPress/gutenberg/pull/64267/commits/c05ef325975bf3f020227e779afdd5b239c088bd